### PR TITLE
spread tests: update gnome extension tests

### PR DESCRIPTION
**Pull request by sergiusens**
Restrict to architectures the content snap is available on.
Clean up summary and use the base logic setup implemented in all the tests.

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

File: tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml [hunk 1]
```diff
@@ -5,7 +5,6 @@ description: This is a basic gnome snap (*only* uses glib). It simply prints a h
 
 grade: devel
 confinement: strict
-base: core18
```
Comment (cjp256) [inline tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml]: The gnome extension works on core i assume?

Comment (sergiusens) [reply tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml]: no, the logic is applied on "prepare"

Comment (kenvandine) [inline tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml]: Why drop the base?  I think in this case it's important.

Comment (sergiusens) [reply tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml]: there's a generic "set-base" call under `prepare`

**Pull request merged**